collector/rpc_metrics.go

```diff
@@ -222,3 +224,21 @@ func (collector *NodeRpcMetrics) Collect(ch chan<- prometheus.Metric) {
 		}
 	}
 }
+
+func (collector *NodeRpcMetrics) RecordValidators() {
+	for {
+		srExt, err := collector.externalClient.Get("status", nil)
+		if err != nil {
+			fmt.Println("Failed to get status from external endpoint.")
+		}
+
+		r, err := collector.externalClient.Get("validators", []uint64{srExt.Status.SyncInfo.LatestBlockHeight - 3})
```

Comment:
What's up with this `srExt.Status.SyncInfo.LatestBlockHeight - 3` part?
Maybe worth a comment explaining

Reply:
yeah thats reasonable. `srExt` should be `statusResponseExternal` which better frames the fact that it is the status response object from the external client